### ISSUE TITLE
Response not valid JSON for simple_chatbot

### ISSUE THREAD

**Issue by robmarkcole** [shroominic/fastui-chat#7]
Hi
I created a Github codespace, installed using `./dev-install.sh` and pip, then:
```
export OPENAI_API_KEY=sk...
cd examples
python simple_chatbot.py
```
No errors in logs but on the page:

<img width="302" alt="image" src="https://github.com/shroominic/fastui-chat/assets/11855322/2ff68636-71fd-4f70-bd7e-5766131b91a7">

**Comment by shroominic** [comment]
ahhh thanks for mentioning I need to add an index page you need to go to the page `/chat`

**Comment by shroominic** [comment]
just released v0.2.1 fixing this :)
